CondFormats/SiPixelObjects/src/classes_def.xml

```diff
@@ -77,8 +77,14 @@
   <class name="SiPixelQualityProbabilities::probabilityVec"/>
   <class name="std::map<unsigned int,SiPixelQualityProbabilities::probabilityVec>"/>
 
-  <class pattern="PixelDCSObject<*>"/> 
-  <class pattern="PixelDCSObject<*>::Item"/> 
-  <class pattern="PixelDCSObject<*>::Type"/> 
```

Comment:
@makortel I notice that this class pattern (with `::Type`) is not explicitated: is it intended?

Reply:
Yes, this is intentional. The `::Type` is an alias for the template argument itself
https://github.com/cms-sw/cmssw/blob/6262bcf9627f03ad2b1c72f28ff2392b6ae6be9c/CondFormats/SiPixelObjects/interface/PixelDCSObject.h#L21-L23
and I opted to include `CaenChannel` explicitly, whereas `bool` and `float` are handled implicitly by ROOT.